shared/Microsoft.Extensions.CommandLineUtils.Sources/Output/ConsoleReporter.cs

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+
+namespace Microsoft.Extensions.Tools.Internal
```

Comment:
Nit: can you move these copied files into tools/KoreBuild.Console/ ? We don't need them in shared/, and also they're not part of Microsoft.Extensions.CommandLineUtils.Sources